src/types/components.ts

```diff
@@ -0,0 +1,18 @@
+import { FileObject } from "./aws";
```

Comment:
as widely used component related type tend to be single for reach component, just group them in this file.